julia/core.py

```diff
@@ -650,6 +657,10 @@ def __init__(self, init_julia=True, jl_init_path=None, runtime=None,
         self.sprint = self.eval('sprint')
         self.showerror = self.eval('showerror')
 
+        if self.eval('VERSION >= v"0.7-"'):
+            self.eval("@eval Main import Base.MainInclude: eval, include")
+            # https://github.com/JuliaLang/julia/issues/28825
```

Comment:
This should fix #202. The reason why it's in this PR is that this fix depends on https://github.com/JuliaPy/pyjulia/pull/200/commits/b908e6bd85664e9b846a80c58710fecb1028d822.